biztalk/install-and-config-guides/configure-biztalk-server.md

```diff
@@ -243,6 +243,32 @@ BAM alerts require BAM tools to be enabled.
     |Windows group|You can keep the default group name, or enter something custom. |
     |Windows SharePoint Services Adapter Web site|Select the Web site that hosts the Windows SharePoint Service Adapter web service.|
 
+### Configure the REST APIs
```

Comment:
Should you mention in some way that these config modules are only available starting with biztalk 2020?

Reply:
suggestion:

### Configure the REST APIs - BizTalk Server 2020 onwards
(this is similar to the way the version is specified for 'Configure Windows SharePoint Services web service')

Reply:
I think there is still something different between here and the sharepoint case.
REST APIs and TMS will be in main stream, i don't think we want to add BizTalk Server 2020 to title in all these cases.
while sharepoint config is out of main stream, that is why it's special.

Reply:
Keep this in the "NOTE" for now, we will see how we want to improve this or not later.